frontend/src/components/gamefield/Gamefield.js

```diff
@@ -32,25 +46,34 @@ function GameField() {
             socket.on('chat message', (message) => {
                 const messageElement = document.createElement('div');
```

Comment:
A chatet is átírhatnánk reactbe ahelyett, hogy getElementById és appendChild-ozunk.